internal/policy/operator/restricted_network_aware.go

```diff
@@ -37,6 +37,8 @@ func (p FollowsRestrictedNetworkEnablementGuidelines) validate(ctx context.Conte
 		return false, err
 	}
 
+	// TODO: This annotation is going to be DEPRECATED, so we should also check for the new `disconnected` specified annotation
```

Comment:
Unrelated change.

Reply:
This is related...when the required annotations check gets enforced, this restricted check will have to be updated to the new annotation format. Is it just a wording problem? I can re-word this to be clearer.

Reply:
After thinking about it, it probably makes more sense for this to live in a GH/JIRA issues for all the things that need to change once the required annotations check is enforced.

Reply:
Created the below to cover this and removed the `TODO`
- #1104